docs/tutorials/getting_started.md

```diff
@@ -286,11 +247,105 @@ Resolve Traefik error in Juju:
 juju resolve traefik/0
 ```
 
+## 6. Deploy the gNodeB and a cellphone simulator
+
+Inside the `terraform` directory create a new module:
+
+```console
+cat << EOF > ran.tf
+resource "juju_model" "ran-simulator" {
+  name = "ran"
+}
+
+module "gnbsim" {
+  source = "git::https://github.com/canonical/sdcore-gnbsim-k8s-operator//terraform"
+
+  model      = juju_model.ran-simulator.name
+  depends_on = [module.sdcore-router]
+}
+
+resource "juju_offer" "gnbsim-fiveg-gnb-identity" {
+  model            = juju_model.ran-simulator.name
+  application_name = module.gnbsim.app_name
+  endpoint         = module.gnbsim.provides.fiveg_gnb_identity
+}
+
+resource "juju_integration" "gnbsim-amf" {
+  model = juju_model.ran-simulator.name
+
+  application {
+    name     = module.gnbsim.app_name
+    endpoint = module.gnbsim.requires.fiveg_n2
+  }
+
+  application {
+    offer_url = module.sdcore.amf_fiveg_n2_offer_url
+  }
+}
+
+resource "juju_integration" "gnbsim-nms" {
+  model = juju_model.sdcore.name
+
+  application {
+    name     = module.sdcore.nms_app_name
+    endpoint = module.sdcore.fiveg_gnb_identity_endpoint
+  }
+
+  application {
+    offer_url = juju_offer.gnbsim-fiveg-gnb-identity.url
+  }
+}
+
+EOF
+```
+
+Initialize Juju Terraform provider:
+
+```console
+terraform init
+```
+
+Apply new configuration:
+
+```console
+terraform apply -auto-approve
+```
+
+Monitor the status of the deployment:
+
+```console
+juju switch ran
+watch -n 1 -c juju status --color --relations
+```
+
+The deployment is ready when the `gnbsim` application is in the `Active/Idle` state.<br>
```

Comment:
Why deploy those in a separate model? We really want to keep this getting started as simple as possible

Reply:
It is as simple as possible.
We need a separate module, because we can't consume a Juju offer in the same module it was created in.